packages/ci-v5/CHANGELOG.md

```diff
@@ -681,7 +681,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [1.5.2] - 2017-04-27
 ### Changed
-- Releax engines requirements in package.json.
+- Relax engines requirements in package.json.
```

Comment:
So projects don't like changing changelogs.

It's also possible some files come from an upstream. In general, I'm happy to try to send changes upstream, but I don't make promises as to when (I have a long queue). Happy to drop if there's an upstream...